scripts/plots/cam/exevs_refs_profile_plots.sh

```diff
@@ -0,0 +1,360 @@
+	       #thresh_fcst='==0.10000'
+	       thresh_fcst='==273.15'
+	       thresh_obs='<273.15'
+             elif [ $VAR =  WIND_ge30kt ] ; then
+	       thresh_fcst='==0.10000'
+	       thresh_obs='>=15.4'
+	     elif [ $VAR =  WIND_ge40kt ] ; then
+	       thresh_fcst='==0.10000'
+               thresh_obs='>=20.58'
+             fi	       
+	 fi
+
+         sed -e "s!model_list!$models!g" -e "s!stat_list!$stat_list!g"  -e "s!thresh_fcst!$thresh_fcst!g" -e "s!thresh_obs!$thresh_obs!g"  -e "s!fcst_init_hour!$fcst_init_hour!g" -e "s!fcst_valid_hour!$fcst_valid_hour!g" -e "s!fcst_lead!$fcst_lead!g" -e "s!interp_pnts!$interp_pnts!g"  $USHevs/cam/evs_refs_plots_config.sh > run_py.${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh
+
+         chmod +x  run_py.${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh
+
+         echo "${DATA}/run_py.${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh" >> run_${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh
+
+         #Save for restart
+         echo "if [ -s ${plot_dir}/${score_type}_regional_*_valid_${fcst_valid_hour}z_*${var_rst}*_${stats_rst}*.png ] ; then" >> run_${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh
+         echo "  cp -v ${plot_dir}/${score_type}_regional_*_valid_${fcst_valid_hour}z_*${var_rst}*_${stats_rst}*.png $restart" >> run_${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh
+	 echo "  >$restart/run_${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.completed" >> run_${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh
+	 echo "fi" >> run_${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh
+
+         chmod +x  run_${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh 
+         echo "${DATA}/run_${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh" >> run_all_poe.sh
+
+       else
+          #Restart from existing png files of previous run
+          cp $restart/${score_type}_regional_*_valid_${fcst_valid_hour}z_*${var_rst}*_${stats_rst}*.png ${plot_dir}/.
+       fi
+
+      done #end of line_type
+
+     done # end of level
+
+    done #end of VAR
+
+   done #end of fcst_lead
+
+  done # end of fcst_valid_hour 
+
+ done #end of score_type
+
+done #end of stats 
+
+chmod +x run_all_poe.sh
+
+#***************************************************************************
+# Run the POE script in parallel or in sequence order to generate png files
+#**************************************************************************
+if [ $run_mpi = yes ] ; then
+   mpiexec -np 60 -depth 1 --cpu-bind verbose,depth cfp ${DATA}/run_all_poe.sh
```

Comment:
Not sure if this matter but having `mpiexec -np 60 -ppn 60 --cpu-bind verbose,depth cfp ${DATA}/run_all_poe.sh` would follow the format of the other ex-scripts I've seen so far.